viz_scripts/scaffolding.py

```diff
@@ -125,18 +127,44 @@ def load_viz_notebook_data(year, month, program, study_type, dic_re, dic_pur=Non
     if "distance" in expanded_ct.columns:
         unit_conversions(expanded_ct)
 
-    # Mapping new mode labels with dictionaries
-    # CASE 2 of https://github.com/e-mission/em-public-dashboard/issues/69#issuecomment-1256835867
-    if "mode_confirm" in expanded_ct.columns:
-        expanded_ct['Mode_confirm']= expanded_ct['mode_confirm'].map(dic_re)
-    if study_type == 'program':
+    # Extract translations_data for dynamic_labels
+    if has_dynamic_labels:
+        print("This json has dynamic label")
+        req = requests.get(dynamic_labels_url)
+        if req.status_code != 200:
+            print("Unable to download dynamic_labels")
+        else:
+            print("dynamic labels download was successful.")
+            dynamic_labels = json.loads(req.text)
+
+        # Extract translations section
+        translations_data = dynamic_labels["translations"]["en"]
```

Comment:
is there a reason why this is here instead of `generate_plots.py`? I can think of at least one reason why having it in `generate_plots.py` is better

Reply:
I felt it'd be appropriate to unfold the dynamic_label's URL in the `scaffolding.py` instead of `generate_plots.py` because we aren't currently using it in `generate_plots.py` and we'd be passing an json file instead of an URL over to the different notebooks. In the case we are passing the json file, I am not quite sure what should we pass into the initial params for the notebook. Please suggest if you'd still like to move this into `generate_plots.py`.

Reply:
you would not pass in a json **file** unless you were planning to pass in the file pointer.
`json.load` returns a json **object**
you can pass in the initial params using

```
custom_labels = {
}
```

or fill it in with 2-3 default values 

Note that you will need to test both the case with custom_labels present and the case with custom_labels absent

Reply:
Moved this code block generate_plots.py. Using dynamic_labels = { } to pass in the initial params.